Rakefile

```diff
@@ -113,6 +119,25 @@ rescue
   puts 'Rubocop not loaded'
 end
 
+namespace :config do
```

Comment:
It should *always* be done. If you look at the original test failure:

```
mkdir /home/travis/build/Katello/katello-installer/_build/parser_cache

kafo-export-params -c config/katello.yaml -f parsercache --no-parser-cache -o /home/travis/build/Katello/katello-installer/_build/parser_cache/katello.yaml

D, [2016-10-27T14:52:16.819596 #3276] DEBUG -- : Using Puppet module parser KafoParsers::PuppetModuleParser

Manifest foreman_proxy/manifests/plugin/ansible.pp was not found in /home/travis/build/Katello/katello-installer/_build/modules, /home/travis/build/Katello/katello-installer/modules
```

Kafo is using the configs to generate the parser cache, but it doesn't appear to be running the migrations (maybe that's a kafo bug? I dunno), but it can't generate the cache against our config because the ansible module isn't in the version of modules we're pulling.